examples/acados_template/python/pendulum_example/generate_c_code.py

```diff
@@ -123,8 +123,10 @@
 # ocp.solver_config.nlp_solver_type = 'SQP_RTI'
 
 # set header path
-ocp.acados_include_path  = '/usr/local/include'
-ocp.acados_lib_path      = '/usr/local/lib'
+# ocp.acados_include_path  = '/usr/local/include'
+# ocp.acados_lib_path      = '/usr/local/lib'
+ocp.acados_include_path  = '~/acados/include'
```

Comment:
can you use something more general here? like `ACADOS_INSTALL_DIR`

Reply:
Yes, I could, but ACADOS_INSTALL_DIR is only set by the MATLAB interface scripts. Is there any environment variable that is set by make install?